app/policies/membership_policy.rb

```diff
@@ -10,6 +10,10 @@ def index?
     return true if person.memberships.pluck(:status).include?('member')
   end
 
+  def set_moderator?
+    true if person.staff?
+  end
```

Comment:
Not blocking: if we change from `set_moderator!` in the model we will want this to reflect the new name.